src/main/java/org/edumips64/webclient.gwt.xml

```diff
@@ -2,6 +2,14 @@
   <inherits name="com.google.gwt.user.User"/>
```

Comment:
User makes almost no sense in a web worker, I advise turning this down to just Core (and also removing at least Logging, since that requires widgets too i believe - JSON ... there is probably a nicer way to do that, i dont know what it requires off the top of my head).

Reply:
Thanks for the suggestion, removed!

This also removed the need for the `user.agent` property to be changed.